### DISCUSSION TITLE
Issue: Time not working well

### DISCUSSION THREAD

**Discussion by RCD800** [f1multiviewer/issue-tracker#90] [Feature requests] [answered]
![image](https://user-images.githubusercontent.com/85562954/177008490-3d26ac8c-6b60-41f3-b054-b8f59aac9259.png)
The track time is not displaying properly (track time was 2 or 3 pm at the moment
![image](https://user-images.githubusercontent.com/85562954/177008497-c337cef0-6ab7-4c97-84da-3b6376ad6c00.png)
The time when you hover the message is fine

**Comment by f1multiviewer** [comment, marked as answer]
Is this still broken? I think this is fixed now :)

**Reply by RCD800** [reply]
Sorry, forgot to close it, working good now :)

**Reply by f1multiviewer** [reply]
No worries, trying to clean the repo up a bit 👍
